fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/AbstractSumoAmbassador.java

```diff
@@ -165,14 +165,13 @@ public abstract class AbstractSumoAmbassador extends AbstractFederateAmbassador
     /**
      * List of vehicles that are simulated externally.
      */
-    private final HashMap<String, ExternalVehicleState> externalVehicleMap = new HashMap<>();
+    protected final HashMap<String, ExternalVehicleState> externalVehicleMap = new HashMap<>();
```

Comment:
can you rename this to `externalVehicles`? I also would prefer to use `Map` for the declaration part.